util/cmake/SeqAnBuildSystem.cmake

```diff
@@ -373,8 +373,8 @@ macro (seqan_setup_library)
             configure_file("util/pkgconfig/seqan.pc.in" "${CMAKE_BINARY_DIR}/util/pkgconfig/seqan-${SEQAN_VERSION_MAJOR}.pc" @ONLY)
             install(FILES "${CMAKE_BINARY_DIR}/util/pkgconfig/seqan-${SEQAN_VERSION_MAJOR}.pc" DESTINATION share/pkgconfig)
         endif (NOT CMAKE_SYSTEM_NAME MATCHES Windows)
-        # Install FindSeqAn TODO(h-2) rename FindSeqAn.cmake to FindSeqAn${SEQAN_VERSION_MAJOR}.cmake after 2.x cycle
-        install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/util/cmake/FindSeqAn.cmake" DESTINATION share/cmake/Modules)
+        # Install FindSeqAn TODO(h-2) rename SeqAnConfig.cmake to SeqAnConfig${SEQAN_VERSION_MAJOR}.cmake after 2.x cycle
+        install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/util/cmake/SeqAnConfig.cmake" DESTINATION share/cmake/)
```

Comment:
Well I will rename it to ```seqan-config.cmake ``` and it should live probably in ```${prefix}/share/cmake/seqan```. Actually both would be sufficient, but it is likely that there is a global cmake folder with all the projects.